src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs

```diff
@@ -54,7 +55,47 @@ public sealed class CSharpCompilationOptions : CompilationOptions, IEquatable<CS
             StrongNameProvider strongNameProvider = null)
             : this(outputKind, false, moduleName, mainTypeName, scriptClassName, usings, optimizationLevel, checkOverflow, allowUnsafe,
                    cryptoKeyContainer, cryptoKeyFile, cryptoPublicKey, delaySign, platform, generalDiagnosticOption, warningLevel,
+                   specificDiagnosticOptions, concurrentBuild, deterministic,
+                   extendedCustomDebugInformation: true,
```

Comment:
Good question.  It looks like this is only happening for `extendedCustomDebugInformation` and `debugPlusMode`.  These were here before I touched the file, but I'll make these changes if you think they should be different.